protocol/v1/qbft/instance/msgcont/inmem/inmem.go

```diff
@@ -138,7 +138,8 @@ func (c *messagesContainer) OverrideMessages(msg *specqbft.SignedMessage, data [
 }
 
 func (c *messagesContainer) PartialChangeRoundQuorum(stateRound specqbft.Round) (found bool, lowestChangeRound specqbft.Round) {
-	lowestChangeRound = specqbft.Round(100000) // just a random really large round number
+	seenLowestChangeRound := false
```

Comment:
optimization?

Reply:
@nivBlox I don't like having a hardcoded value like `100000`, so this is to get rid of a hardcoded value